src/entry_point.py

```diff
@@ -85,11 +85,17 @@ def __call__(self, parser, namespace, values, option_string=None):
                 num = None  # let the multiprocessing module decide
             setattr(namespace, self.dest, num)
 
-    p = argparse.ArgumentParser(description="constructor helper subcommand")
+    # Remove "constructor" so that it does not clash with the uninstall subcommand
+    del sys.argv[1]
+    p = argparse.ArgumentParser(
+        prog="conda.exe constructor", description="constructor helper subcommand"
+    )
+    # Cannot make this a required argument or `conda.exe constructor uninstall --prefix`
+    # will not work (would have to be `conda constructor --prefix uninstall`)
```

Comment:
We are checking against `prefix=None` later right? So in practice it is required for the user. A comment here to clarify would be perfect.